src/Core/System/Consent/DTO/ConsentStateLogRecord.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Consent\DTO;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\Consent\ConsentStatus;
+
+/**
+ * @codeCoverageIgnore
+ */
+#[Package('data-services')]
+readonly class ConsentStateLogRecord
+{
+    public function __construct(
+        public ConsentStatus $status,
+        public ?string $identifier,
```

Comment:
why can the identifier be null? Does that make sense? Same for the consent log as well

Especially as in the event it is not null (however i see in the DB schema it is also nullable)